ts/components/services/ServiceSectionListComponent.tsx

```diff
@@ -0,0 +1,93 @@
+import * as pot from "italia-ts-commons/lib/pot";
+import * as React from "react";
+import {
+  ListRenderItemInfo,
+  RefreshControl,
+  SectionList,
+  SectionListData,
+  StyleSheet
+} from "react-native";
+
+import { H3, ListItem } from "native-base";
+import { ServicePublic } from "../../../definitions/backend/ServicePublic";
+import { ProfileState } from "../../store/reducers/profile";
+import variables from "../../theme/variables";
+import { ServiceListItem } from "./ServiceListItem";
+
+type OwnProps = {
+  // Can't use ReadonlyArray because of the SectionList section prop
+  // typescript definition.
+  // tslint:disable-next-line:readonly-array
+  sections: Array<SectionListData<pot.Pot<ServicePublic, Error>>>;
+  profile: ProfileState;
+  refreshing: boolean;
+  onRefresh: () => void;
+  onSelect: (service: ServicePublic) => void;
+};
+
+type Props = OwnProps;
+
+const styles = StyleSheet.create({
+  listItem: {
+    paddingLeft: variables.contentPadding,
+    paddingRight: variables.contentPadding
+  }
+});
+
+/**
+ * A component to render a list of services grouped for organization.
+ */
+class ServiceSectionListComponent extends React.Component<Props> {
+  private renderServiceItem = (
+    itemInfo: ListRenderItemInfo<pot.Pot<ServicePublic, Error>>
+  ) => (
+    <ServiceListItem
+      item={itemInfo.item}
+      profile={this.props.profile}
+      onSelect={this.props.onSelect}
+    />
+  );
+
+  private getServiceKey = (
+    potService: pot.Pot<ServicePublic, Error>,
+    index: number
+  ): string => {
```

Comment:
avoid the return if it's the only statement

```suggestion
  ): string => pot.getOrElse(
```